operations/client.go

```diff
@@ -152,3 +155,35 @@ func getJWTFromKanopy() cli.Command {
 		},
 	}
 }
+
+func setJWTInConfig() cli.Command {
+	return cli.Command{
+		Name:    "set-jwt",
+		Usage:   "generate a JWT token through kanopy-oidc and set it as \"jwt\" in the evergreen config file",
+		Aliases: []string{"sjwt"},
+		Action: func(c *cli.Context) error {
+			jwt, err := runKanopyOIDCLogin()
+			if jwt == "" {
+				return errors.New("no JWT received - ensure kanopy-oidc is installed and configured correctly")
+			}
+			if err != nil {
+				return errors.Wrap(err, "running kanopy-oidc login")
+			}
+
+			confPath := c.Parent().String(confFlagName)
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "loading configuration")
+			}
+
+			conf.JWT = jwt
+
+			if err := conf.Write(""); err != nil {
+				grip.Warning(message.WrapError(err, "failed to set a jwt token "))
```

Comment:
I think this should return the error, since the operation should be considered failed otherwise (e.g. if some other program has a lock on the file, this might fail I think). This would better support retries